manifests/scenarios/gmp-otel/1_collector.yaml

```diff
@@ -190,34 +187,86 @@ data:
+              target_label: namespace
+              action: replace
+            - target_label: job
+              replacement: avalanche
+              action: replace
+            - source_labels: [__meta_kubernetes_pod_phase]
+              regex: (Failed|Succeeded)
+              action: drop
+            - target_label: project_id
+              replacement: {{ .Env.PROJECT_ID }}
+              action: replace
+            - target_label: location
+              replacement: {{ .Env.ZONE }}
+              action: replace
+            - target_label: cluster
+              replacement: {{ .Env.CLUSTER_NAME }}
               action: replace
-              target_label: __metrics_path__
-              regex: (.+)
-            - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+            - source_labels: [__meta_kubernetes_pod_name]
+              target_label: __tmp_instance
               action: replace
-              regex: (.+):(?:\d+);(\d+)
+            - source_labels: [__meta_kubernetes_pod_controller_kind, __meta_kubernetes_pod_node_name]
+              regex: DaemonSet;(.*)
+              target_label: __tmp_instance
+              replacement: $$1
+              action: replace
+            - source_labels: [__meta_kubernetes_pod_container_port_name]
+              regex: metrics
+              action: keep
+            - source_labels: [__tmp_instance, __meta_kubernetes_pod_container_port_name]
+              regex: (.+);(.+)
+              target_label: instance
               replacement: $$1:$$2
-              target_label: __address__
-            - action: labelmap
-              regex: __meta_kubernetes_pod_label_(.+)
+              action: replace
+            kubernetes_sd_configs:
+            - role: pod
+              kubeconfig_file: ""
+              follow_redirects: true
+              enable_http2: true
+              selectors:
+              - role: pod
+                field: spec.nodeName=$(env:NODE_NAME)
+          #            kubernetes_sd_configs:
+          #            - role: pod
+          #            relabel_configs:
+          #            - source_labels: [__meta_kubernetes_pod_label_app]
+          #              action: keep
+          #              regex: avalanche
+          #            - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
+          #              action: replace
+          #              target_label: __metrics_path__
+          #              regex: (.+)
+          #            - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+          #              action: replace
+          #              regex: (.+):(?:\d+);(\d+)
+          #              replacement: $$1:$$2
+          #              target_label: __address__
+          #            - action: labelmap
+          #              regex: __meta_kubernetes_pod_label_(.+)
 
-      prometheus/self-metrics:
```

Comment:
I removed this as I don't want to send this data to GMP, I want to scrape otel collector /metrics page. Do I need to configure something? 

I see some metrics like `otelcol_process_memory_rss` but no `otelcol_receiver_...` metrics. 🤔 Any ideas?

cc @ridwanmsharif